railties/lib/rails/generators/rails/app/templates/Gemfile.tt

```diff
@@ -66,7 +66,11 @@ end
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem "capybara", ">= 3.26"
+<%- if RUBY_VERSION >= "3.0.0" -%>
+  gem "selenium-webdriver", ">= 4.0.0.rc1"
```

Comment:
Since this is for Rails 7, we can just move the requirement to be `>= 4.0.0.rc1` and remove the need to check the ruby version.

Reply:
Ok! Thus, I am going to remove the test too.